skimage/measure/fit.py

```diff
@@ -573,6 +573,10 @@ def ransac(data, model_class, min_samples, residual_threshold,
         where the probability (confidence) is typically set to a high value
         such as 0.99, and e is the current fraction of inliers w.r.t. the
         total number of samples.
+    exceptions : exception class or tuple of exception classes
```

Comment:
Hm. You are right. But without breaking backwards compatibility, I am not sure how to approach this.

`_center_and_normalize_points` throws a ZeroDivision exception for degenerate cases now (unfortunately, this was not the case before).

What do you think?